### PR TITLE
Automatically set direction into html tag

### PR DESCRIPTION
**Pull request by BaseMax**
Hi Folks,

We have to recognize current language is **LTR** or **RTL**.

`http://127.0.0.1:3000/?locale=en`

### EN
```
<!DOCTYPE html>
<html lang="en" dir="ltr">
```

`http://127.0.0.1:3000/?locale=fa`

### FA
```
<!DOCTYPE html>
<html lang="fa" dir="rtl">
```

------

Brian was the one who suggested this module and I  adjust and add it.

cc @bdougie: Thanks for mention this module.
Refer to: https://github.com/github/opensourcefriday/issues/585

Correct me if I made any mistakes.

Regards,
Max

### CODE REVIEW

**Comment by deniseyu** [comment]
hey @BaseMax, I think you'll also need to run `bundle install` locally so the versions are locked into the `Gemfile.lock`, then commit those changes as well!

**Comment by BaseMax** [comment]
Hey Denise,

Thanks for pointing me to this. I had completely forgotten. @deniseyu 

Please let me know if there is any other case.

**Comment by BaseMax** [comment]
Hi, Can you review? @bdougie @MikeMcQuaid

**Review by deniseyu — APPROVED** [review]
Thanks for updating the Gemfile!

File: Gemfile [hunk 1]
```diff
@@ -46,3 +46,5 @@ group :test do
   gem "simplecov", require: false
   gem "vcr", require: false
 end
+
+gem "rtl"
```
Comment (MikeMcQuaid) [inline Gemfile:50]: Can this be moved to the list of gems at the top of the file?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="<%= I18n.locale %>">
+<html lang="<%= I18n.locale %>" dir="<%= (Rtl.rtl? I18n.locale) ? 'rtl' :'ltr' %>">
```
Comment (MikeMcQuaid) [inline app/views/layouts/application.html.erb:2]:
```suggestion
<html lang="<%= I18n.locale %>" dir="<%= (Rtl.rtl? I18n.locale) ? "rtl" : "ltr" %>">
```

File: Gemfile (hunk 1 above)
Comment (BaseMax) [reply Gemfile]: Sure, yeah.

**Review by BaseMax — COMMENTED** [review]
Hi,
I made the necessary corrections. @MikeMcQuaid 
Please take another look at it. Thanks

**Comment by MikeMcQuaid** [comment]
Thanks again @BaseMax!

**Pull request merged**